src/ConsoleApplication.php

```diff
@@ -35,17 +35,22 @@ public static function boot(string $name = 'Tempest', ?AppConfig $appConfig = nu
 
         $container = $kernel->init();
 
+        $argumentBag = new ConsoleArgumentBag($_SERVER['argv']);
```

Comment:
Did some cleanup, and registered the argument bag in the container for later use. We might need to clean this up further, but it didn't feel _super important_ for now.